src/main/java/me/zeroeightsix/kami/command/commands/MacroCommand.kt

```diff
@@ -48,18 +50,15 @@ class MacroCommand : Command("macro", ChunkBuilder().append("command", true, Enu
                 }
                 sendChatMessage("'&7$rKey&f' has the following macros: ")
                 Macro.sendMacrosToChat(keyList.toTypedArray())
-                return
```

Comment:
when statement only runs the first branch that is true, then it will run the line after the statement. And there is nothing after the when statement. So it is safe to remove